Core/src/test/java/com/plotsquared/core/plot/FlagTest.java

```diff
@@ -74,4 +75,48 @@ public void testFlagName() {
         Assertions.assertEquals("use", flagName);
     }
 
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleSingularAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test\" \"\"").getValue();
+            Assertions.assertEquals("test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title =  PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleMultipleWords() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"a very long subtitle\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("a very long subtitle", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleEmptyAndSubTitleSingleWords() {
```

Comment:
The naming here is confusing, it's multiple words in the sub title

Reply:
fixed by using "single" as word.

https://github.com/IntellectualSites/PlotSquared/pull/3410/commits/b03fc7df804561c5cd86982a0be511a488b893ac#diff-d97a8ca9b5c7b0ab2416ccc3d57f70cc4a522a71bcfb8b3b696310ccc5605d4bR110